plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -216,6 +213,16 @@ public static Object[][] timestampTypes()
         };
     }
 
+    @DataProvider
+    public Object[][] testTableAndColumNameDataProvider()
+    {
+        //we should filter Upper case column names
+        Object[][] filteredUpperCaseColumnNameProvider = Arrays.stream(testColumnNameDataProvider())
+                .filter(objects -> Arrays.stream(objects).anyMatch(o -> !String.valueOf(o).matches(".*[A-Z].*")))
+                .toArray(Object[][]::new);
+        return concat(filteredUpperCaseColumnNameProvider, testTableNameDataProvider());
```

Comment:
The concat here will make addressing the above `//TODO move test to BaseConnectorTest and merge this data provider with testColumnNameDataProvider` harder.
A person doing that will need to think why table names have more test cases than column names, and what do with that.

Which elements from `testTableNameDataProvider()`  actually add more value compared with what `testColumnNameDataProvider()` already provides?

Reply:
I add the comment like this:
`//TODO merge TableNameDataProvider and ColumnNameDataProvider to ObjectNameDataProvider in BaseConnectorTest`

So I expect that person will move the test to BCT and also merge this 2 data provider in one common ObjectNameDataProvider and then just applied 2 filters to column and table names. 
I think it could break a lot in other connectors, maybe lets do it in follow up PR and now just don't touch BCT

Reply:
> So I expect that person will move the test to BCT and also merge this 2 data provider in one common ObjectNameDataProvider 

i still don't know why would i merge the list instead of using the longer (better?) one everywhere.

Which elements from testTableNameDataProvider() actually add more value compared with what testColumnNameDataProvider() already provides?

Reply:
By merging I mean that person will take 2 DataProviders, replace it with one ObjectNameDataProvider and replace 2 lists of special characters with one big list with all cases, which will be used everywhere